app_flutter/lib/status_grid.dart

```diff
@@ -45,29 +44,128 @@ class StatusGridContainer extends StatelessWidget {
           );
         }
 
+        final Map<String, int> taskColumnKeyIndex =
+            createTaskColumnKeyIndex(statuses);
+        final List<List<Task>> taskMatrix =
+            createTaskMatrix(statuses, taskColumnKeyIndex);
+        final List<Task> taskIconRow = createTaskIconRow(statuses, taskColumnKeyIndex);
+
         return StatusGrid(
           statuses: statuses,
+          taskMatrix: taskMatrix,
+          taskIconRow: taskIconRow,
         );
       },
     );
   }
+
+  /// A unique index for grouping [Task] from separate [CommitStatus].
+  ///
+  /// [task.stageName] and [task.name] are not unique. However, they
```

Comment:
Good idea!

Reply:
I just want to point out Github does not support reacting with the smiley emoji. What a tragedy :P